File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.java
// Last updated: 8/1/2025, 1:58:31 PM
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length==0){
            return 0;
        }
        int i=0;
        for(int j=1;j<nums.length;j++){
            if(nums[j] !=nums[i]){
                i++;
                nums[i]=nums[j];
            }
        }
        return i+1;
    }
}

File: 26-RemoveDuplicatesfromSortedArray/26-RemoveDuplicatesfromSortedArray.py
# Last updated: 1/7/2026, 8:26:10 PM
1class Solution:
2    def removeDuplicates(self, nums: List[int]) -> int:
3        k=1
4
5        if not nums:
6            return 0
7
8        for i in range(1,len(nums)):
9            if nums[i]!=nums[i-1]:
10                nums[k]=nums[i]
11                k+=1
12
13        return k
14            
15        